### ISSUE TITLE
Failed on Linux Mint 20 Cinnamon

### ISSUE THREAD

**Issue by spartan** [geerlingguy/ansible-role-docker#229]
With the following playbook:

```yml
---
- hosts: 127.0.0.1
  become: true
  roles:
    - geerlingguy.docker
```

I receive the error:

```
TASK [geerlingguy.docker : Add Docker apt key.] *********************************************************************************************************************************************************************************************
fatal: [127.0.0.1]: FAILED! => {"changed": false, "msg": "Failed to download key at https://download.docker.com/linux/linux mint/gpg: An unknown error occurred: URL can't contain control characters. '/linux/linux mint/gpg' (found at least ' ')"}
...ignoring
```

Followed by:

```
ASK [geerlingguy.docker : Add Docker apt key (alternative for older systems without SNI).] *************************************************************************************************************************************************
fatal: [127.0.0.1]: FAILED! => {"changed": true, "cmd": "curl -sSL https://download.docker.com/linux/linux mint/gpg | sudo apt-key add -\n", "delta": "0:00:00.335674", "end": "2020-09-18 02:29:05.866236", "msg": "non-zero return code", "rc": 2, "start": "2020-09-18 02:29:05.530562", "stderr": "Warning: apt-key output should not be parsed (stdout is not a terminal)\ncurl: (6) Could not resolve host: mint\ngpg: no valid OpenPGP data found.", "stderr_lines": ["Warning: apt-key output should not be parsed (stdout is not a terminal)", "curl: (6) Could not resolve host: mint", "gpg: no valid OpenPGP data found."], "stdout": "", "stdout_lines": []}
```

**Comment by 1stDimension** [comment]
Hello!

## tl;dr 
override variables
- docker_apt_gpg_key 
- docker_apt_repository
just as if you were using Ubuntu. That is
```yaml
docker_apt_gpg_key: https://download.docker.com/linux/ubuntu/gpg
docker_apt_repository: "deb [arch=amd64] https://download.docker.com/linux/ubuntu <<ver >> {{ docker_apt_release_channel }}"
```
replace **<< ver >>** with version of Ubuntu Mint 20 is based on. I believe it's *focal*.
Your playbook should look similar to this 

```yaml
---
- hosts: 127.0.0.1
  become: true
  roles:
    - role: geerlingguy.docker
      vars:
        docker_apt_gpg_key: https://download.docker.com/linux/ubuntu/gpg
        docker_apt_repository: "deb [arch=amd64] https://download.docker.com/linux/ubuntu focal {{ docker_apt_release_channel }}"
```

## Longer explanation 

This role adds repository and gpg keys of Docker's repos. The problem is that docker's repos don't serve packages for
linux mint. In this case Docker docs advice to use packages of the distro your distro is based on (Ubuntu). 

Docker docs [here](https://docs.docker.com/engine/install/) say
> Likewise, users of Ubuntu derivatives such as “Kubuntu”, “Lubuntu” or “Xubuntu” should follow the installation instructions for Ubuntu, substituting the version of their distro for the corresponding Ubuntu release. Refer to the documentation of your distro to find which Ubuntu release corresponds with your derivative version.

**Comment by spartan** [comment]
Thank you!
